README.md

```diff
@@ -1,8 +1,23 @@
 # Testify — Android Screenshot Testing
 
+> **Note**
```

Comment:
Should this be a warning? I was actually addressing this issue before I noticed you threw up a PR. Also maybe slap this right at the very top since it's critical information.

Had something like this. <img width="1050" alt="Screen Shot 2022-08-05 at 3 04 00 PM" src="https://user-images.githubusercontent.com/105017/183144388-677303d1-84d5-461c-a9b4-bb7ec4de39f2.png">

Reply:
I had `warning`, but then changed it to `Note` because I didn't like the look of the `Warning` style ...  It was purely a preference thing, but I'll change it back.

+1 I agree with moving the notice to the very top